.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
         stages: [commit-msg]
     -   id: check-flake8
         name: Check Flake8
-        language: script
-        entry: scripts/travisci/check_flake8.sh
+        language: system
```

Comment:
is it not sufficient to pin the flake8 version both here and in setup.py?

Reply:
Do you mean without mentioning a language? It's a mandatory key for pre-commit and `system` seems like the most relevant value. `python` could work but it tries to install flake-docstrings in a virtual env and fails on the pydocstyle issue.

Reply:
i mean not using system at all (just the default pre-commit rule), but pinning the version to match ours.

Reply:
We don't pin flake8 version yet, but we could do something like this

```

-   repo: https://gitlab.com/pycqa/flake8
    rev: '3.7.8'
    hooks:
    -   id: flake8
        stages: [commit, push]
        additional_dependencies: ['flake8-import-order', 'flake8-docstrings==1.3.0', 'pydocstyle<4.0.0']
```

Is this what you meant?